extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -14,23 +14,112 @@
  * limitations under the License.
  */
 
+.i-amphtml-story-interactive-slider-container {
+  --percentage: 0%;
+}
+
 .i-amphtml-story-interactive-slider-input-container {
+  display: flex !important;
   background: var(--i-amphtml-interactive-option-background-color) !important;
   border-radius: 2em !important;
   box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
 }
 
+.i-amphtml-story-interactive-slider-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0);
+  width: 2.8em;
+  height: 2.8em;
+  top: 50%;
+  left: var(--percentage);
+  text-align: center;
+  line-height: 2.6em;
+  z-index: 2 !important;
+  font-weight: 500 !important;
+  color: #fff !important;
+  transition: transform 0.3s ease-in-out !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble::before {
+  text-align: center;
+  line-height: 2.8em;
+}
+
+.i-amphtml-story-interactive-slider-bubble.show {
+  transform: translate(-50%, -170%) scale(1) !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble::after {
+  position: absolute !important;
+  content: '' !important;
+  box-sizing: border-box !important;
+  height: 2.8em !important;
+  width: 2.8em !important;
+  background: var(--interactive-accent-color) !important;
+  z-index: -1 !important;
+  left: 50% !important;
+  top: -0.1em !important;
+  transform: translateX(-50%) rotate(45deg) !important;
+  border-bottom-left-radius: 50% !important;
+  box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25) !important;
+  border-top-left-radius: 50% !important;
+  border-top-right-radius: 50% !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble {
+  cursor: initial !important;
+  font-weight: 700 !important;
+  transform: translate(-50%, -50%) scale(0.8) !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble::after {
+  cursor: initial !important;
+  border: 0.125em solid #fff !important;
+  border-bottom-right-radius: 50% !important;
+}
+
+.i-amphtml-story-interactive-slider-field {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+  height: 100% !important;
+  width: 100% !important;
+  margin: 0em 1em !important;
+  position: relative !important;
+}
```

Comment:
Do we use this now? Or can it be removed?

Reply:
Yes it positions the numbers, as well as the track. Without it no track appears, nor can users interact with the slider

Reply:
I think the `...-field` container might be unnecessary, can you check?